Content.Shared/_RMC14/Entrenching/EntrenchingToolSystem.cs

```diff
@@ -48,10 +50,62 @@ private void OnAfterInteract(Entity<EntrenchingToolComponent> tool, ref AfterInt
         if (!args.CanReach)
             return;
 
+        if (HasComp<BarricadeSandbagComponent>(args.Target))
+        {
+            DismantleSandbagBaricade(tool, ref args);
+            args.Handled = true;
+            return;
+        }
+
         StartDigging(tool, args.User, args.ClickLocation);
         args.Handled = true;
     }
 
+    private void DismantleSandbagBaricade(Entity<EntrenchingToolComponent> tool, ref AfterInteractEvent args)
+    {
+        if (TryComp(tool, out ItemToggleComponent? toggle) && !toggle.Activated)
+            return;
+
+        _popup.PopupClient(Loc.GetString("cm-entrenching-dismantle"), args.User, args.User);
+
+        var ev = new SandbagDismantleDoAfterEvent(GetNetCoordinates(args.ClickLocation));
```

Comment:
This should be try get net coordinates with a return and maybe a log warning, so that if a client passes a malicious input it doesn't error

Reply:
There is no `TryGetNetCoordinates` method in `EntitySystem` or anywhere else i could find, only a `TryGetNetEntity` which wouldn't work here as far as i can tell. There are two other instances of just `GetNetCoordinates` being used within the same file before i had made any changes.

I'm going to mark this as resolved but if you want i can make changes to all three instances i just need to know how.

Reply:
Nevermind, I checked now and apparently I hallucinated that method